build.go

```diff
@@ -144,7 +144,7 @@ func packageGrafana() {
 	grunt(postProcessArgs...)
 	pkgArch = previousPkgArch
 
-	if goos == "linux" && goarch == "amd64"{
+	if goos == "linux" {
```

Comment:
side note : to prevent building packages on non arm64, test should be made on `pkgArch`, not `goarch` (which is always native host, so arm64, when running `build.go`). So condition `goarch == "amd64"` had no effect.

Reply:
Yeah, that was a bit of an oversight. But no we won't need it :)a